Lib/test/test_asyncio/test_events.py

```diff
@@ -2861,6 +2861,19 @@ class TestCGetEventLoop(GetEventLoopTestsMixin, unittest.TestCase):
         get_running_loop_impl = events._c_get_running_loop
         get_event_loop_impl = events._c_get_event_loop
 
+class TestServer(unittest.TestCase):
+
+    def test_get_loop(self):
+        loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(loop)
+        proto = MyProto(loop)
+        server = loop.create_server(lambda: proto, '0.0.0.0', 0)
```

Comment:
The method is a coroutine, use `server = loop.run_until_complete(loop.create_server(lambda: proto, '0.0.0.0', 0))`